src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -1333,6 +1467,17 @@ internal void CheckOpenLocked()
             });
         }
 
+        internal bool HasScopeNoneCache(string scope)
```

Comment:
Awkward naming, I had to look at the implementation to see what it does.

Reply:
renamed.